administration/src/components/verification/VerificationController.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button, Card, H1, Spinner } from '@blueprintjs/core'
```

Comment:
i think everything connected to the application process should be responsible and use material components @michael-markl ?

Reply:
So not using a width for the Json View already improved the responsiveness a lot. However it is probably best to improve this by using mui components.

Reply:
I agree... Also afaik, JsonView does not use blueprint a lot (I think only for the Attachment Download Button). We could probably reuse a lot of it if we switch to MUI.